routes/profileRoutes.js

```diff
@@ -0,0 +1,17 @@
+import { Router } from 'express';
+
+import AuthMiddlewware from '../middlewares/Authentication';
+import ProfileController from '../controllers/ProfileController';
+import Validate from '../middlewares/inputValidation';
+
+const { createProfile, editProfile, viewProfile } = ProfileController;
+
+const { verifyToken } = AuthMiddlewware;
+const router = Router();
+
+router.post('/', verifyToken, Validate.createProfileValidate, createProfile);
+router.patch('/', verifyToken, Validate.editProfileValidate, editProfile);
```

Comment:
Shouldn't this route have an `/:id` to specify the particular resource you are editing?